content/app/accessing-cloud-infra.md

```diff
@@ -54,9 +54,13 @@ To be able to impersonate the above service account, annotate your service accou
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: NAME
+  name: NAME #for the above examples this should be "sa" 
```

Comment:
Could we actually change this to be something like `myfirsttenancy-sa` it'll read easier and be clearer that this is meant to change by tenancy

Reply:
Heh - so counter-intuitively this needs to be "sa" and not e.g. "myfirsttenancy/sa"  - basically these accounts are all (or need to be) created per (sub)-namespace and I didn't really understand that from the docs. 

Here is the example that I actually got working: https://github.com/coreeng/nathan-mono-repo2/blob/main/nathan-go-app-2/helm-charts/app/templates/service-account.yaml#L4 

And it took me _a lot_ of messing round with other names (e.g. "tenancy/sa" or ca etc.) before I realised it was just as simple as using the name after the "/". These are all namespace(/subname space) scoped things (as I understand it) so no need for a globally unique name.